docs/source/bot.rst

```diff
@@ -0,0 +1,71 @@
+Telegram Bot
+============
+
+You can add new torrents with Telegram bot. Here described how to configure and run it.
```

Comment:
We need to mention what Telegram library we use. Maybe even add that into ``extras`` section of ``setup.py``.